config-local.json

```diff
@@ -1,17 +1,9 @@
 {
-  "publicKeyFile": "./public-key.json",
-  "secretKeyFile": "./secret-key.json",
   "client_id": "dcdb5ae7add825d2",
   "client_secret": "b93ef8a8f3e553a430d7e5b904c6132b2722633af9f03128029201d24a97f2a8",
   "redirect_uri": "http://127.0.0.1:8080/api/oauth",
-  "auth_uri": "http://127.0.0.1:9010/v1/authorization",
-  "oauth_uri": "http://127.0.0.1:9010/v1",
-  "content_uri": "http://127.0.0.1:3030",
-  "profile_uri": "http://127.0.0.1:1111/v1",
-  "scopes": "profile",
-  "preverify_email_jku": "http://127.0.0.1:8080/.well-known/public-keys",
-
+  "issuer_uri": "https://rfkelly.dev.lcip.org",
```

Comment:
@rfk what should the local value be here?

Reply:
Oh whoops, this must have snuck in accidentally...